src/scripts/sync_synonym.py

```diff
@@ -247,7 +252,7 @@ def sync_synonyms(
     mondo_exclusion_configs: Union[Path, str], onto_synonym_types_path: Union[Path, str],
     mondo_mappings_path: Union[Path, str], onto_config_path: Union[Path, str], outpath_added: Union[Path, str],
     outpath_confirmed: Union[Path, str], outpath_updated: Union[Path, str],
-    outpath_combined: Union[Path, str], outpath_deleted: Union[Path, str] = None,
+    outpath_combined: Union[Path, str], outpath_deleted: Union[Path, str] = None, doid_added_filtration=False
```

Comment:
This line `outpath_combined: Union[Path, str], outpath_deleted: Union[Path, str] = None, doid_added_filtration=False` also looks like an accident.

Reply:
This one is also a new addition at the end of the line: `, doid_added_filtration=False`

Reply:
ok, i did not realize that removing the combined file was only in https://github.com/monarch-initiative/mondo-ingest/pull/767